src/stores/Annotation/Annotation.js

```diff
@@ -1274,12 +1275,40 @@ export const Annotation = types
 
     acceptSuggestion(id) {
       const item = self.suggestions.get(id);
+      let itemId = id;
+      const isGlobalClassification = item.classification;
+
+      if (isFF(FF_LLM_EPIC)) {
```

Comment:
add a comment, please, that this piece of code prevents from creating duplicated global classifications